tests/components/network_sediment_transporter/test_init_sediment_pulser.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+from landlab.components.network_sediment_transporter.sediment_pulser_at_links import (
+    SedimentPulserAtLinks,
+)
+from landlab.components.network_sediment_transporter.sediment_pulser_each_parcel import (
+    SedimentPulserEachParcel,
+)
```

Comment:
Use relative imports.

Reply:
Sorry to be dense @mcflugen but because this file is in the 'tests' folder and we're importing from 'components', I don't see how a relative import would have a benefit...?

Reply:
@pfeiffea You are quite correct. Sorry about that. Just to clean things up a bit, how about
```python
from landlab.components import SedimentPulserAtLinks, SedimentPulserEachParcel
```

Reply:
@mcflugen sounds great!